src/main/java/core/basesyntax/Main.java

```diff
@@ -29,8 +32,9 @@ public static void main(String[] args) {
         TransactionDto transactionDto = new TransactionDto();
         ReportMaker reportMaker = new ReportMaker();
         ReadFromFile readFromFile = new ReadFromFile();
+
         WriteToFile writeToFile = new WriteToFile();
-        List<String> data =  readFromFile.read(FROM_FILE_NAME);
+        List<String> data = readFromFile.read(FROM_FILE_NAME);
         transactionDto.convertFromListToDb(data, operationStrategyMap);
         String report = reportMaker.make(Storage.fruits);
```

Comment:
you use Storage directly as you have DAO. I'd choose either way. Either you go to DAO and then dao uses storage or
you use storage without dao everywhere. Definitely I am not sure 100%

Reply:
Agree